### ISSUE TITLE
Cannot see resolution switcher on Live Stream

### ISSUE THREAD

**Issue by DrDoug88** [WWBN/AVideo#5921]
I'm not sure if this is intentional or not, but I am unable to see the resolution switcher on the video player with a live stream. I have implemented the VideoHLS plugin as well as the VideoResolutionSwitcher plugin. The nginx .conf has been updated to split the live into the three different resolutions, and I can confirm that it is generating the four different stream directories (low, med, hi, src) and that each directory is being updated with .ts files, etc. I also have the Adaptive option selected with the Live plugin.

So I know it is working correctly on the NGINX side. What I cannot confirm is that it is working correctly on the output side. I do not see a resolution switching option (the gear icon) on the video player. I can see the option on videos uploaded via the encoder and with different resolutions (btw, where is the option to change what types of resolutions are created when using the encoder?).

From what I understand, the adaptive option on the Live plugin will automatically determine what resolution to send, based on the speed/bandwidth of the connection. I feel like the adaptive part is working, since turning it off will cause the stream to sit there and spin, while with it enabled, it loads right up (there is about 1,400 miles between the client machine and the video server, thus the connection issue is expected)

Given all of this, is there still not a gear option within the player to manually switch the resolution of the live stream? Is this intentional? Is there a way to see the gear option, i.e. by some other configuration of options?

Thanks in advance.

**Comment by DanielnetoDotCom** [comment]
Hi, 

I am pretty sure we never had a resolution switcher for live streams, only for VOD

**Comment by DrDoug88** [comment]
Okay, thanks Daniel. One other question, where is the option to change what types of resolutions are created when using the encoder?

**Comment by DanielnetoDotCom** [comment]
Here

![chrome-capture (3)](https://user-images.githubusercontent.com/18009293/140405223-09068ac6-c823-4277-ae74-1363c61ecebf.gif)

**Comment by DrDoug88** [comment]
I only see Sharing Queue and Queue Log at the top. I tried logging in with a regular user and site admin account, but still only see those 2 tabs.

![image](https://user-images.githubusercontent.com/80653458/140405902-c1a8eaa1-e51e-4d86-be99-66f9a520b716.png)

**Comment by DanielnetoDotCom** [comment]
Try this https://github.com/WWBN/AVideo/issues/5225

**Comment by DrDoug88** [comment]
I didn't have the Encoder install files. I did a git pull and attempted to get the Encoder directory in the right place. Anyhow, long story short, I completely forgot about the encoder database. I ended up just updating the admin user in the encoder database to have admin rights. The admin username was changed awhile back to enforce the unique email policy. So, I have updated it, and now I have proper admin access to the encoder site, and I can see all of the options correctly. Thanks for your help Daniel.
